### DISCUSSION TITLE
PSP Launcher

### DISCUSSION THREAD

**Discussion by valk7** [aldostools/webMAN-MOD#659] [General]
Hi Aldo. 
I just want to know why PSP Remaster launcher looks like homebrew error? and PSP Launcher can't be uninstalled. It can be uninstalled after uninstalling webman mod.

**Comment by aldostools** [comment]
They look ok when you mount a PSP game.
The launchers are redirected to an empty folder to prevent that the system hangs in the user launch them without mounting a game.

Use http://0/remap.ps3 from the PS3 Internet browser to unlock them without mounting a game. That option is also in webMAN Games > webMAN Setup menu as Unmap PS3.

**Comment by valk7** [comment]
Oh I thought it was an error from the psp launcher.
So it's just a remap so that when running psp games it doesn't hangs.

Can i request something
can you separate the Neo Geo Pocket ROMS folder with Neo Geo Pocket Color. Those are two different ROMs but can run on the same core.
there are also Wonderswam and Wonderswam Color ROMS, can you separate that Folder too.

maybe like this:
/ROMS/NGP for Neo Geo Pocket
/ROMS/NGPC for Neo Geo Pocket Color
/ROMS/WSWAM for Wonderswam
/ROMS/WSWAMC for Wonderswam Color

**Reply by aldostools** [reply]
Update to the latest build to get support for these folders.
